js/runner.go

```diff
@@ -441,13 +441,13 @@ func (u *VU) RunOnce(ctx context.Context) error {
 func (u *VU) runFn(
 	ctx context.Context, group *lib.Group, isDefault bool, fn goja.Callable, args ...goja.Value,
 ) (goja.Value, bool, time.Duration, error) {
-	cookieJar, err := cookiejar.New(nil)
-	if err != nil {
-		return goja.Undefined(), false, time.Duration(0), err
-	}
-
-	if u.Runner.Bundle.Options.NoCookiesReset.Valid && u.Runner.Bundle.Options.NoCookiesReset.Bool {
-		cookieJar = u.CookieJar
+	cookieJar := u.CookieJar
+	if !u.Runner.Bundle.Options.NoCookiesReset.ValueOrZero() {
```

Comment:
What do you think about renaming this option (in the next breaking change release) to `PersistCookies`/`K6_PERSIST_COOKIES`? `K6_NO_COOKIES_RESET` sounds a bit awkward to me, mainly because of the negation and "persist cookies" is a bit clearer IMO.

Reply:
I would argue it will probably be part of the next HTTP API ;)

Reply:
Hhmm true, though a global option would still be useful for defining the default behavior. Maybe we should model it closer to browsers: run in a global mode of either persisting or clearing cookies per iteration (similar to having the option of "private" mode in browsers), while allowing the setting to be overriden via the API (so per request).

Anyway, out of scope for this PR. :)